postgres/src/client.rs

```diff
@@ -263,10 +263,11 @@ impl Client {
     /// # Ok(())
```

Comment:
You should be able to update the example above to remove the `.map(|p| ...)` bit.